sites/aviationpros.com/server/components/nodes/content-list.marko

```diff
@@ -17,11 +17,12 @@ $ const { linkAttrs } = input;
   image-position=imagePosition
   attrs=input.attrs
 >
-  <if(displayImage)>
+  <if(displayImage && primaryImage.src)>
```

Comment:
The second condition shouldn't be necessary now that you're disabling placeholders

Reply:
@zarathustra323 - dropped that check.